backend/core/database.py

```diff
@@ -509,6 +535,29 @@ def config_request_rpc(self):
                 while self.rules is None:
                     self.connection.drain_events()
 
+        def signal_loading(self, status=False):
```

Comment:
database can directly call the query, no need to send a message to itself right?

Reply:
fixed